comment/views.py

```diff
@@ -40,37 +56,37 @@ def delete(self, request, comment_id):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
     def put(self, request, comment_id):
+        try:
+            request.COOKIES.get('access_token')
+        except:
+            return Response({"detail": "Authentication credentials not provided"}, status=status.HTTP_401_UNAUTHORIZED)
+        author = request.user
+        if not author.is_authenticated:
+            return Response({"detail": "Permission denied"}, status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
```suggestion
        user = request.user
        content = request.data.get('content')
        
        if not user.is_authenticated:
            return Response({"detail": "Authentication credentials not provided"}, status=status.HTTP_401_UNAUTHORIZED)
        
        if not content:
            return Response({"detail": "missing fields ['content']"}, status=status.HTTP_400_BAD_REQUEST)
        
        try:
             comment = Comment.objects.get(id=comment_id)
         except:
             return Response({"detail": "Not found."}, status=status.HTTP_404_NOT_FOUND)
             
        if comment.author != user:
            return Response({"detail": "Permission denied"}, status=status.HTTP_401_UNAUTHORIZED)
```
comment의 author과 로그인한 user를 비교하는 부분이 빠져서 예시 코드를 적어보았습니다!